src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -49,13 +49,6 @@ public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBas
         public List<AzureSqlDatabaseModel> Database { get; set; }
 
         /// <summary>
-        /// Gets or sets the tags associated with the Azure SQL Database Failover Group
```

Comment:
@ziwa-msft server side deprecation cannot be aligned at the same time. Powershell deprecation has a process and it needs to handled in a separate way.
Please add obsolteattributes to whatever needs to be deprecated.